lib/portlayer/attach/communication/lazy.go

```diff
@@ -50,3 +50,11 @@ func (l *LazySessionInteractor) Initialize() (SessionInteractor, error) {
 	}
 	return l.si, nil
 }
+
+// Cleanup returns either an initialized connection, or nil if it was never initialized
+func (l *LazySessionInteractor) Cleanup() (SessionInteractor, error) {
```

Comment:
Might be worth rebasing - I seem to remember these changes in a different PR (@hickeng's) that I suspect has already been merged.

Reply:
It looks like this picked up a bunch of random CLs from other people. Doing a force push seems to have cleared that up.